scripts/clean-shrinkwrap.js

```diff
@@ -15,29 +15,258 @@
 
 'use strict';
 
+const _ = require('lodash');
+const fs = require('fs');
 const path = require('path');
-const os = require('os');
 const packageJSON = require('../package.json');
```

Comment:
We only need to `require` the package.json, however we need to shrinkwrap path in two places: a `require` call and the output of `fs.writeFile`, so better to extract it as a constant to avoid path issues (`require()` has special logic to understand `./` on Windows, but its not safe to assume that for `fs.writeFile` as well)